packages/mui-material/src/MenuList/MenuList.test.js

```diff
@@ -44,6 +46,20 @@ describe('<MenuList />', () => {
 
       expect(getAllByRole('menuitem')).to.have.length(2);
     });
+
+    it('should not add tabIndex to presentation elements like Divider when all menuItems are disabled', () => {
+      const { getByRole } = render(
+        <MenuList>
+          <MenuItem disabled>one</MenuItem>
+          <MenuItem disabled>two</MenuItem>
+          <MenuItem disabled>three</MenuItem>
+          <Divider />
+          <MenuItem disabled>four</MenuItem>
+        </MenuList>,
+      );
+
+      expect(getByRole('separator')).to.not.have.attribute('tabIndex');
```

Comment:
To follow convention:
```suggestion
      expect(getByRole('separator')).not.to.have.attribute('tabIndex');
```